src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -693,8 +696,8 @@ h2.location a {
 	flex-grow: 1;
 	margin: 0px;
 	padding: 0px;
-	overflow-wrap: break-word;
```

Comment:
The use of `break-word` followed by `anywhere` is intentional. See https://github.com/rust-lang/rust/blob/a39bdb1d6b9eaf23f2636baee0949d67890abcd8/src/librustdoc/html/static/css/rustdoc.css#L107-L112.

Reply:
Oh interesting! I'll put the comment here too then.